js/gated-content/src/views/CategoriesListingPage.vue

```diff
@@ -83,21 +82,6 @@ export default {
     };
   },
   computed: {
-    selectedType() {
```

Comment:
The categories controller now doesn't need the type and bundle filters separately.
And the `selectedBundle` was moved to the `FilterAndSortMixin` since it is needed also for the `CategoryPage`